### ISSUE TITLE
most used tree-renderers

### ISSUE THREAD

**Issue by arlin** [OpenTreeOfLife/phylostylotastic#23]
FYI, there is a biorxiv preprint with a list of citation counts for a bunch of tree viewers (http://biorxiv.org/content/early/2015/12/25/035360.full.pdf+html) . 

![screen shot 2016-12-01 at 11 22 24 am](https://cloud.githubusercontent.com/assets/1659198/20801957/755b70ca-b7b8-11e6-9fef-2c865fe171e2.png)

**Comment by jhcepas** [comment]
@arlin 
this is the ongoing work I mentioned in one of our meetings. The rooting tests are now expanded to more tree viewers and toolkits. The citation table is also updated here: 
http://biorxiv.org/content/early/2016/09/07/035360

![image](https://cloud.githubusercontent.com/assets/163658/20802287/e83749ba-b7eb-11e6-89ee-d069e04bd978.png)

**Comment by rvosa** [comment]
It's interesting, but right away I am skeptical:

- who ever cites a tree viewer?
- for online viewers, do users realize it is a separate artefact
independent of the web page they are viewing it in?
- some tools are not just viewers, e.g. archaeopteryx can also be used on
the command line without viewing anything
- also, I don't believe that dendroscope us so popular - but if it is, we
should try to get the buggy nexml implementation fixed

@rdmpage must be delighted about the popularity of treeview, though :-)

**Comment by rdmpage** [comment]
@rvosa Yep, the TreeView paper is the only paper that gives my citation count any credibility...

**Comment by jimallman** [comment]
Just leaving a note here about Elsevier's [interactive tree feature](https://www.elsevier.com/authors/author-services/enrichments/interactive-phylogenetic-trees), since it came up at the Chicago workshop. This is one of [a large family of "enrichments"](https://www.elsevier.com/authors/author-services/enrichments) for online articles, and is based on [jsPhyloSVG](http://www.jsphylosvg.com/). (It's unclear whether this relationship has anything to do with [that project's comatose state](http://iphylo.blogspot.co.uk/2012/12/elsevier-articles-have-interactive.html), as it's been inactive since 2012.)

**Comment by BenStoever** [comment]
I just wanted to add [TreeGraph 2](http://treegraph.bioinfweb.info/) to this list because I'm interested to support TSS there, as discussed earlier.

Since this issue is about being "most used" here are some stats of TreeGraph 2: 470 citations in Web of Science, 655 in Google scholar, ~15 downloads per day, one update check by running applications to our servers every ~15 minutes.

I will look into supporting TSS in the future, because to me it seems makes sense for the following reasons:

1. TreeGraph is currently refactored to use NeXML as its main format and we are looking for the best way to store its various tree formatations.
2. Being format-independent using the "virtual DOM" concept of TSS would allow to apply externally defined formats to trees in all different formats that are supported by TreeGraph. The concept of "virtual DOM" fits very well to the abstraction over various tree formats that is provided by [JPhyloIO](http://bioinfweb.info/JPhyloIO/) and used in TreeGraph.
3. The conditional formatting options like `.node[support < 0.5] { size: support * 10px; }` are very similar to the features of TreeGraph to automatically set [distance values](http://treegraph.bioinfweb.info/Help/wiki/Set_distance_values_by_node/branch_data) or [colors](http://treegraph.bioinfweb.info/Help/wiki/Set_colors_by_node/branch_data) by annotations.
4. Users in the past already requested to have external format classes (as found, e.g., in MS Word) that can be applied to new documents, so they do not have to repeat applying their formats to every new tree. TSS could also provide a solution for this.

**Comment by BenStoever** [comment]
@rvosa 
To your question "who ever cites a tree viewer?" In the case of TreeGraph 2, mainly people who used it to create a figure for their publication or used its feature to combine support values from alternative analyses usually cite it in the methods part. Not everyone does this, but many do.

This maybe only true for tree editors and for sole tree viewers the fraction of users the cite the software may be even smaller. Generally, I think one can assume that the number of users is much higher than the number of citations, but it probably correlates with some linear factor. (Users like students or high school teachers use it, but do not necessarily produce published work with it.)

**Comment by rvosa** [comment]
@BenStoever 

>  In the case of TreeGraph 2, mainly people who used it to create a figure for their publication

You're probably right: you make a figure, add it to your paper, and then wonder what to put in the caption and the methods to explain how you made it. I guess this gives a citation advantage to tools that make 'publication ready' visualizations in vector format, as opposed to data exploration tools that only produce bitmaps (by making screenshots, for example).
